Library/Formula/git-lfs.rb

```diff
@@ -1,9 +1,9 @@
 class GitLfs < Formula
   desc "Git extension for versioning large files"
   homepage "https://github.com/github/git-lfs"
-  url "https://github.com/github/git-lfs/archive/v0.5.4-homebrew.tar.gz"
-  version "0.5.4"
-  sha256 "cf860d56d86cda40a5b883b81f4d991617788c13d5e09be6b35c8bb9ac878ad7"
+  url "https://github.com/github/git-lfs/archive/v0.6.0.tar.gz"
+  version "0.6.0"
```

Comment:
You can cut this line now. Was only needed because the previous version had the `-homebrew` appended to it.